src/qml/QFieldLocalDataPickerScreen.qml

```diff
@@ -808,6 +817,54 @@ Page {
         }
       }
     }
+
+    Menu {
+      id: selectionMenu
+      x: parent.width - width - 8
+      width: determineMenuWidth(selectionMenu)
+
+      MenuItem {
+        id: uploadToWebdav
+
+        enabled: table.selectedItemsWebDavConfigured
+        visible: enabled
+
+        font: Theme.defaultFont
+        width: parent.width
+        height: enabled ? 48 : 0
+        leftPadding: Theme.menuItemLeftPadding
+
+        text: qsTr("Upload file(s) to WebDAV")
+        onTriggered: {
+          console.log("Send it to web dav!");
+        }
+      }
```

Comment:
TODO? :) Do you want to do it in this PR or merge this and work on the WebDAV-specific bits in a different PR?